ably/docs
# [WEB-367] Add internal links on the website Socket.io concept

Explain why this change is being made.
----
Links to Socket.io concept page was not present in the content of the website

Explain how this is accomplished.
----
Added Socket.io links to content of the textile files


How do you manually test this?
----
Navigate to `/concepts/feathersjs`
Navigate to `/concepts/websockets`

Look for `Socket.io` links should navigate to `/concepts/socketio`

Merge/Deploy Checklist
----


Frontend Checklist
----

- [ ] Have you compared with the initial design / fit our [brand guidelines](https://brand.ably.com/)?
